operational_analysis/toolkits/power_curve/parametric_forms.py

```diff
@@ -32,7 +32,7 @@ def logistic5param(x, a, b, c, d, g):
         Function[pandas.Series[real]] -> pandas.Series[real]
 
     """
-    return d + (a - d) / (1 + (x / c) ** b) ** g
+    return np.where(x!=0.0, d + (a - d) / (1 + (x / c) ** b) ** g, d)
```

Comment:
when x == 0, this should return a, not d, since the curve asymptotically approaches a as x approaches 0.

Reply:
wait, never mind, it approaches a when the slope parameter is positive. and d when negative.

Reply:
If we restrict b>0 then the issue goes away. This method was originally implemented from the following citation: https://www.sciencedirect.com/science/article/pii/S1364032113007296?via%3Dihub

I checked and it does not seem to place any restriction on the sign of b.

Reply:
ok, I would suggest something like an if else statement. If b >= 0: use the original line since there is no problem, else: use your new code that fixes the divide by zero issue. 

This way both positive and negative values of b are handled.